File: src/Amelia.Domain/Models/Sprint.cs
using System;
using System.Collections.Generic;

namespace Amelia.Domain.Models
{
    public class Sprint : IEntityBase
    {
        public int Id {get;set;}
        public string Name{get;set;}
        public string Description{get;set;}
        public DateTime StartDate{get;set;}
        public DateTime DueDate{get;set;}
        public DateTime CreatedOn{get;set;}
        public DateTime UpdatedOn{get;set;}
        public Project Project{get;set;} 

        public virtual IEnumerable<Task> Tasks{get;set;}
    }
}

File: src/Amelia.WebApp/wwwroot/lib/spa/modules/backlog/domain/sprint.js
"use strict";
var Sprint = (function () {
    function Sprint(id, name, projectId, description, startDate, dueDate, createdOn, updatedOn) {
        this.Id = id;
        this.Name = name;
        this.Description = description;
        this.ProjectId = projectId;
        this.StartDate = startDate;
        this.DueDate = dueDate;
        this.CreatedOn = createdOn;
        this.UpdatedOn = updatedOn;
    }
    return Sprint;
}());
exports.Sprint = Sprint;
